### DISCUSSION TITLE
關於作業二 答案設置 是否有誤

### DISCUSSION THREAD

**Discussion by Rebecca111419** [1101-GS4719-C/Python-Community-GS4719-C#1692] [General]
作業二題目是要求金字塔圖row = 5,8,10
但答案好像設錯了，設成5,6,9
想確認~~

**Comment by WeiYen1103** [comment]
題目是要寫出能符合題目要求rows的金字塔，建議你不要想著題目只會出rows是多少去解題，換成是想，若rows為多少時圖會怎麼樣， 6 7 8...的話會是如何去解題~~~建議你可以先用VS跑過，可以有結果且正確再丟上去會比較好debug喔

**Comment by Rebecca111419** [comment]
想請問 我前面是這樣寫，顯示錯誤，但我看不出我錯在哪邊，不知如何debug....

def hw2_0(rows=3):
    output = ''
    # ======↓請在這個區域寫程式↓=====
    for i in range(0, rows):
        if i != (rows-1):
            output += " " * (rows - i-1) + " *" * (i + 1) + '\n'
        else:
            output += " " * (rows - i-1) + " *" * (i + 1)

    # ======↑請在這個區域寫程式↑=====
    return output

**Comment by 1chooo** [comment]
這是正確答案的輸出
```
    *
   * *
  * * *
 * * * *
* * * * *
```

以下是我執行你程式的輸出
```
     *
    * *
   * * *
  * * * *
 * * * * *
```

看起來你每一行的最左邊都多了一個 "空格" 可以往這方面去試試看！

**Reply by Rebecca111419** [reply]
噢噢 好 我試試，大感謝!!!

**Reply by Rebecca111419** [reply]
謝謝你，我後來答案正確了~~ (遲交兩天希望影響不大(??
